pkgs/by-name/xf/xfel/package.nix

```
@@ -0,0 +1,36 @@
+{
+  fetchFromGitHub,
+  lib,
+  libusb1,
+  pkg-config,
+  stdenv,
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xfel";
+  version = "1.3.2";
+
+  src = fetchFromGitHub {
+    owner = "xboot";
+    repo = pname;
+    rev = "v${version}";
```

```suggestion
    tag = "v${version}";
```
New preferred argument for tags. `rev` only for commits